src/init.c

```diff
@@ -21,18 +21,22 @@
 
 */
 
+#include <Rdefines.h>
 #include <R_ext/Rdynload.h>
 #include "xxhash.h"
 #include "pmurhash.h"
 
+SEXP R_spookydigest(SEXP s, SEXP fun);
+
 static const R_CallMethodDef callMethods [] = {
     { "PMurHash32", (DL_FUNC) &PMurHash32, 3 },
+    { "R_spookydigest", (DL_FUNC) &R_spookydigest, 2 },
     { NULL, NULL, 0 }
 };
 
 void R_init_digest(DllInfo *info) {
     R_RegisterCCallable("digest", "PMurHash32", (DL_FUNC) &PMurHash32);
-
+    R_RegisterCCallable("digest", "R_spookydigest", (DL_FUNC) &R_spookydigest);
```

Comment:
I know I sent these changes to you. It is not clear we need this.  I do think we had external uses or at least an illustration for `PRMurHash32`.  I think if we kept it I'd want to change the name of `R_spookydigest` to something more standard.  Maybe just `spookydigest_impl` would do.